### ISSUE TITLE
is env is rw

### ISSUE THREAD

**Issue by FCO** [Scimon/p6-Trait-Env#3]
Does this write on the env var?
```perl6
class :: { has $.bla is env is rw }.new.bla = “something”
```

**Comment by Scimon** [comment]
No. The class is populated from the environment but after that it's a copy.

**Comment by FCO** [comment]
Couldn’t it be a Proxy with a write on the env var on STORE

**Comment by Scimon** [comment]
I'll have a think about it, not sure how to tie it in though. If you want to add a patch for it I'll take a look.
